### PR TITLE
feat: #211 Create Textarea component

### PR DESCRIPTION
**Pull request by Shustovsky**

### CODE REVIEW

File: mw-webapp/src/component/textarea/Textarea.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  tags: ["autodocs"],
+};
+
+export default meta;
+
+/**
+ * Default story for the Textarea component.
+ */
+export const Default = () => {
+  const [text, setText] = useState<string>("");
```
Comment (tritonJS826) [inline mw-webapp/src/component/textarea/Textarea.stories.tsx:17]: get rid of value (use default value instead)

File: mw-webapp/src/component/textarea/Textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+/**
+ * Textarea component
+ */
+export const Textarea = (props: TextareaProps) => {
+  const textareaRef = useRef<HTMLTextAreaElement | null>(null);
+
+  useEffect(() => {
+    const currentTextarea = textareaRef.current;
```
Comment (tritonJS826) [inline mw-webapp/src/component/textarea/Textarea.tsx:38]: why do we choose this behavior  for textarea?

File: mw-webapp/src/component/textarea/Textarea.stories.tsx (hunk 1 above)
Comment (Shustovsky) [reply mw-webapp/src/component/textarea/Textarea.stories.tsx:17]: Done

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/component/textarea/Textarea.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <textarea
+      className={clsx(styles.textarea, props.className)}
+      placeholder={props.placeholder}
+      value={value}
+      onChange={handleTextChange}
+      rows={props.rows}
+    />
+  );
+};
```
Comment (Howl404) [inline mw-webapp/src/component/textarea/Textarea.tsx:62]: I think Viktar meaned to delete state and only set defaultValue to input

**Review by Howl404 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/textarea/Textarea.tsx (hunk 3 above)
Comment (Shustovsky) [reply mw-webapp/src/component/textarea/Textarea.tsx:62]: Hmm... that's a strange remark. If Victor had something other, then he will write it by himself)))

Comment (Howl404) [reply mw-webapp/src/component/textarea/Textarea.tsx:62]: Do we really need to have state inside textarea? I think we will use it with onChange function 100% of the time

**Review by Howl404 — APPROVED** [review]

**Pull request merged**